src/components/Visualizations/Table/HeroImage.jsx

```diff
@@ -50,6 +51,15 @@ const TableHeroImage = ({
           role="presentation"
           className={styles.image}
         />
+        {leaverStatus !== undefined && leaverStatus > 1 &&
+          <img
+            src="/assets/images/dota2/disconnect_icon.png"
+            role="presentation"
+            className={styles.abandoned}
+            data-hint={strings[`leaver_status_${leaverStatus}`]}
```

Comment:
Nope. It has the question mark pointer when you mouse over it, but no tooltip. I'm thinking it might be an issue with it being `position: absolute`